pyximages/README.md

```diff
@@ -1,6 +1,7 @@
 # Content of directory pyximages
 
-This directory contains files related to schematic drawings in the Scipy
+This directory contains files related to schematic drawings in the
+Scientific Python
```

Comment:
Here, "Scientific Python Lecture Notes" is all capitalized; elsewhere, "lecture notes" is not. Although it's reasonable to say that the "lecture notes" belong to "Scientific Python", capitalizing everything makes it ambiguous whether the notes are those of "Scientific Python" or "scientific Python". We should decide one way on this and keep it consistent.